### PR TITLE
Mfw 3694 eos   board info on dash should show cbl/ind instead of custom

### PR DESCRIPTION
**Pull request by smitropoulos-arista**

### CODE REVIEW

File: sync-settings/files/product-board-name.init [hunk 1]
```diff
@@ -4,56 +4,84 @@
...
+		exit
+	fi
+
+	SERIAL=$(cat /etc/config/serial | tr -d '[ \t\r\n]')
+	# Q series serial pattern CTWYYWWSSSS
+	# YYWWSSSS is a series of digits representing the year, number of weeks, and a sequence number
+	qSerialPattern="^CTW([1-9][0-9])(0[1-9]|[1-4][0-9]|5[0-3])([0-9]{4})$"
+
+	# EOS additions
+	if [ -f /etc/Eos-release ]; then
```
Comment (AlistairJohnParker) [inline sync-settings/files/product-board-name.init:19]: This is correct upper/lowercase mix ?

File: sync-settings/files/product-board-name.init [hunk 2]
```diff
@@ -4,56 +4,84 @@
...
+				exit 0
+				;;
+			esac
+			;;
+		esac
+
+		# Not exiting here, since we want VMs to overwrite.
+	fi
+
+	case $(board_name) in
```
[the remark this answers is not in the document]
Comment (AlistairJohnParker) [reply sync-settings/files/product-board-name.init:47]: i guess so since this was here before.

[the remark this answers is not in the document]
Comment (smitropoulos-arista) [reply sync-settings/files/product-board-name.init:47]: It's under /lib/functions.sh. I thought of adding the check for EOS there as well, but it's in the OpenWRT repo, didn't really like to add something for another OS there

File: sync-settings/files/product-board-name.init (hunk 1 above)
Comment (smitropoulos-arista) [reply sync-settings/files/product-board-name.init:19]: Yes, although I don't like it either lol

**Pull request merged**